src/horizon_api.ts

```diff
@@ -72,6 +72,7 @@ export namespace Horizon {
     is_authorized: boolean;
     is_authorized_to_maintain_liabilities: boolean;
     is_clawback_enabled: boolean;
+    sponsor?: string;
```

Comment:
This is the important change, addressing [this comment](https://github.com/stellar/js-stellar-sdk/pull/693#discussion_r701346251).